### ISSUE TITLE
Fix S2094 FP: Should not raise for messages

### ISSUE THREAD

**Issue by denis-troller** [SonarSource/sonar-dotnet#9063]
### Description

It is not an uncommon pattern to have messages (commands, events) with absolutely no members in it when using messaging of any kind (Fluxor, MassTransit, Wolverine...) and especially in distributed environment. 
In such cases, rule S2094 raises.

### Repro steps

Consider the following classes, which are totally valid messages and events.

```
public class PlayCommand {}

public class StreamingServiceStartedEvent {}

```

### Expected behavior

I would expect rule S2094 not to raise on such classes.

I would suggest ignoring classes that have known suffixes such as 
- Command
- Event
- Message

**Comment by Tim-Pohlmann** [comment]
[RSPEC PR](https://github.com/SonarSource/rspec/pull/3867)

**Comment by fraserkyle** [comment]
Can you add `Query` suffix to this in order to cover CQRS design pattern please.

**Comment by CristianAmbrosini** [comment]
Hi @fraserkyle! Can I ask you to open a new issue with your request? My colleagues will take care of it as soon as possible
